### PR TITLE
Add Danish (da) translations.

### PR DESCRIPTION
**Pull request by frederikspang**
Todo:
- [ ] Get a few danish reviews.
- [ ] Determine if "Added" etc. within change-types should be translated ([swedish is not](https://keepachangelog.com/sv/1.0.0/)
- [ ] Should we translate v1.0.0 as well? How huge are changes from v1.0.0 to v1.1.0? I think, it depends on estimated release-date of v1.1.0?

### CODE REVIEW

File: source/da/1.1.0/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+    Den nuværende version af GitHub releases er også, diskutérbart,
+    ikke særligt synlig for slutbrugere, ikke lig de typiske
+    kapitaliserede filer (<code>README</code>,
+    <code>CONTRIBUTING</code>, osv.). Et andet mindre problem, er
+    at interfacet ikke tilbyder links til commit logs mellem hvert
+    release.
+
+  %h4#automatic
+    %a.anchor{ href: "#automatic", aria_hidden: "true" }
+    Kan changelogs læses fortolkes automatisk?
```
Comment (emilbaekdahl) [inline source/da/1.1.0/index.html.haml]: "læses fortolkes"?

File: source/da/1.1.0/index.html.haml [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+
+  %p
+    Det er besværligt, fordi folk følger vidt forskellige formater,
```
Comment (emilbaekdahl) [inline source/da/1.1.0/index.html.haml]:
```suggestion
    Det er besværligt, fordi folk følger vidt forskellige formater
```

File: source/da/1.1.0/index.html.haml [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+    og fil-navne.
```
Comment (emilbaekdahl) [inline source/da/1.1.0/index.html.haml]:
```suggestion
    og filnavne.
```

File: source/da/1.1.0/index.html.haml [hunk 4]
```diff
@@ -0,0 +1,341 @@
...
+
+  %p
+    #{link_to "Vandamme", vandamme} er en Ruby gem, lavet af
```
Comment (emilbaekdahl) [inline source/da/1.1.0/index.html.haml]:
```suggestion
    #{link_to "Vandamme", vandamme} er en Ruby gem lavet af
```

File: source/da/1.1.0/index.html.haml [hunk 5]
```diff
@@ -0,0 +1,341 @@
...
+    holdet bag Gemnasium, som læser og fortolker mange (men ikke
+    alle) open source projekters changelogs.
+
+  %h4#yanked
+    %a.anchor{ href: "#yanked", aria_hidden: "true" }
+    Hvad med tilbagetrukkede udgivelser?
+
+  %p
+    Tilbagetrukkede udgivelser er versioner, der er blevet trukket
+    tilbage på grund af en seriøs fejl, eller sikkerhedsbrist. Ofte
```
Comment (emilbaekdahl) [inline source/da/1.1.0/index.html.haml]:
```suggestion
    tilbage på grund af en seriøs fejl eller sikkerhedsbrist. Ofte
```

File: source/da/1.1.0/index.html.haml [hunk 6]
```diff
@@ -0,0 +1,341 @@
...
+    vises disse versioner slet ikke i changelogs. Det bør de. Dette
+    er hvordan du bør vise dem:
+
+  %p <code>## 0.0.5 - 2014-12-13 [YANKED]</code>
+
+  %p
+    Notationen <code>[YANKED]</code>, er højlydt af en årsag. Det er
```
Comment (emilbaekdahl) [inline source/da/1.1.0/index.html.haml]:
```suggestion
    Notationen <code>[YANKED]</code>, er højlydt af en årsag: Det er
```

File: source/da/1.1.0/index.html.haml [hunk 7]
```diff
@@ -0,0 +1,341 @@
...
+
+
+  %h4#rewrite
+    %a.anchor{ href: "#rewrite", aria_hidden: "true" }
+    Bør man nogensinde omskrive en changelog?
+
+  %p
+    Klart! Der er altid gode grunde til at forbedre en changelog.
+    Jeg åbner jævnligt pull-requests, der tilføjer manglende releases
+    til open source projekter, uden en vedligeholdt changelog.
```
Comment (emilbaekdahl) [inline source/da/1.1.0/index.html.haml]:
```suggestion
    til open source projekter uden en vedligeholdt changelog.
```

File: source/da/1.1.0/index.html.haml [hunk 8]
```diff
@@ -0,0 +1,341 @@
...
+
+  %p
+    Det er også muligt, at du vil opdage at du har glemt at addressere
+    en ødelæggdne rettelse i noterne for en version. Det er selvfølgelig
```
Comment (emilbaekdahl) [inline source/da/1.1.0/index.html.haml]:
```suggestion
    en ødelæggende rettelse i noterne for en version. Det er selvfølgelig
```

File: source/da/1.1.0/index.html.haml [hunk 9]
```diff
@@ -0,0 +1,341 @@
...
+    vigtigt at du opdateret en changelog i disse tilfælde.
```
Comment (emilbaekdahl) [inline source/da/1.1.0/index.html.haml]:
```suggestion
    vigtigt at du opdaterer en changelog i disse tilfælde.
```

File: source/da/1.1.0/index.html.haml [hunk 10]
```diff
@@ -0,0 +1,341 @@
...
+
+
+  %h4#contribute
+    %a.anchor{ href: "#contribute", aria_hidden: "true" }
+    Hvordan kan jeg hjælpe til?
+
+  %p
+    Dette dokument er ikke <strong>sandheden</strong>; det er min
+    velovervejede mening, sammen med information og eksempler, jeg har
```
Comment (emilbaekdahl) [inline source/da/1.1.0/index.html.haml]:
```suggestion
    velovervejede mening sammen med information og eksempler jeg har
```

File: source/da/1.1.0/index.html.haml [hunk 11]
```diff
@@ -0,0 +1,341 @@
...
+    Det er fordi, jeg gerne vil have at vores fællesskab når en konsensus.
+    Jeg mener at diskussionen er lige så vigtig som slutresultatet.
+  %p
+    Så, <strong>#{link_to "vær med!", gh}</strong>
+
+.press
+  %h3 Samtaler
+  %p
+    Jeg var med i #{link_to "The Changelog podcast", thechangelog} for
+    at snakke om hvorfor vedligeholdere og bidragsydere bør bekymre sig
```
Comment (emilbaekdahl) [inline source/da/1.1.0/index.html.haml]:
```suggestion
    at snakke om, hvorfor vedligeholdere og bidragsydere bør bekymre sig
```

File: source/da/1.1.0/index.html.haml [hunk 12]
```diff
@@ -0,0 +1,341 @@
...
+    om changelogs, og også om motivationen bag dette projekt.
```
Comment (emilbaekdahl) [inline source/da/1.1.0/index.html.haml]:
```suggestion
    om changelogs og også om motivationen bag dette projekt.
```

**Comment by frederikspang** [comment]
Thanks @emilbaekdahl - Fixed the bunch of reviews and squashed.

**Comment by olivierlacan** [comment]
@frederikspang Do you feel good about the state of this PR?

**Comment by frederikspang** [comment]
I'll give it a lookover for an update at some point soon, if anything has changed! :)

**Comment by olivierlacan** [comment]
@frederikspang Checking in for updates and potential reviews.

**Comment by olivierlacan** [comment]
I'm going to merge this and wait on Danish speakers to chime in and contribute if there are improvements to be made. 😄

**Comment by olivierlacan** [comment]
> * Should we translate v1.0.0 as well? How huge are changes from v1.0.0 to v1.1.0? I think, it depends on estimated release-date of v1.1.0?

I wouldn't worry about that. I need to fix the bug that hides 1.1.0

**Pull request merged**